[Automation] java run fails for org.fusesource.hawtjni:hawtjni-runtime:1.11

## java run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131138
Last 300 lines of the log:
```console
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava
> Task :checkTestTimeoutAnnotations
> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.fusesource.hawtjni:hawtjni-runtime:1.9 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 1.11 failed with exit code 1.
SOFT-FAIL[native-image run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean test -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 846ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 955ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 833ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :processTestResources NO-SOURCE
> Task :compileTestJava
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

1 test completed, 1 failed

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 3 executed, 10 up-to-date

BUILD FAILED in 2s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 1.11 failed with exit code 1.
FAILED[java run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
```

## java run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131127
Last 300 lines of the log:
```console
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :jar
> Task :checkTestTimeoutAnnotations
> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.fusesource.hawtjni:hawtjni-runtime:1.9 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 1.11 failed with exit code 1.
SOFT-FAIL[native-image run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean test -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 850ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 936ms
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 818ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:processResources NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :processTestResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 3 executed, 10 up-to-date

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 1.11 failed with exit code 1.
FAILED[java run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
```

## java run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: java run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131130
Last 300 lines of the log:
```console
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :jar
> Task :compileTestJava
> Task :checkTestTimeoutAnnotations
> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
14 actionable tasks: 4 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.fusesource.hawtjni:hawtjni-runtime:1.9 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 1.11 failed with exit code 1.
SOFT-FAIL[native-image run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.11" ./gradlew clean test -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 849ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileTestJava

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 1s
10 actionable tasks: 1 executed, 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 2s
11 actionable tasks: 3 executed, 8 up-to-date
========================================================================================
 org.fusesource.hawtjni:hawtjni-runtime:1.9:1.11 stage java run
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 842ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :javaTest
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :checkTestTimeoutAnnotations
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:processResources NO-SOURCE
> Task :tck-test-support:classes UP-TO-DATE
> Task :tck-test-support:jar UP-TO-DATE
> Task :compileTestJava
> Task :processTestResources NO-SOURCE
> Task :testClasses

> Task :test FAILED

LibraryTest > loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() FAILED
    java.lang.AssertionError: 
    Expecting actual:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    to contain exactly (and in same order):
      ["META-INF/native/linux64/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/linux/libhawtjni_runtime_coverage_probe_missing.so",
        "META-INF/native/libhawtjni_runtime_coverage_probe_missing.so"]
    but some elements were not expected:
      ["META-INF/native/linux64/amd64/libhawtjni_runtime_coverage_probe_missing.so"]
        at org_fusesource_hawtjni.hawtjni_runtime.LibraryTest.loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails(LibraryTest.java:30)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.9/build/reports/tests/test/index.html

* Try:
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 2s
13 actionable tasks: 3 executed, 10 up-to-date

> Task :javaTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javaTest'.
> Java tests failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'java run' for 1.11 failed with exit code 1.
FAILED[java run][1.11][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="1.11" ./gradlew clean javaTest -Pcoordinates="org.fusesource.hawtjni:hawtjni-runtime:1.9"]
```

--- a/metadata/org.fusesource.hawtjni/hawtjni-runtime/1.11/reachability-metadata.json
+++ b/metadata/org.fusesource.hawtjni/hawtjni-runtime/1.11/reachability-metadata.json
@@ -1,0 +1,10 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "org.fusesource.hawtjni.runtime.Library"
+      },
+      "type": "java.lang.String[]"
+    }
+  ]
+}

--- a/metadata/org.fusesource.hawtjni/hawtjni-runtime/index.json
+++ b/metadata/org.fusesource.hawtjni/hawtjni-runtime/index.json
@@ -1,6 +1,15 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "1.11",
+    "tested-versions" : [
+      "1.11"
+    ],
+    "allowed-packages" : [
+      "org.fusesource.hawtjni.runtime"
+    ]
+  },
+  {
     "metadata-version" : "1.9",
     "source-code-url" : "https://repo1.maven.org/maven2/org/fusesource/hawtjni/hawtjni-runtime/$version$/hawtjni-runtime-$version$-sources.jar",
     "repository-url" : "https://github.com/fusesource/hawtjni",

--- a/metadata/org.fusesource.hawtjni/hawtjni-runtime/index.json
+++ b/metadata/org.fusesource.hawtjni/hawtjni-runtime/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "1.11",
+    "source-code-url" : "https://repo1.maven.org/maven2/org/fusesource/hawtjni/hawtjni-runtime/$version$/hawtjni-runtime-$version$-sources.jar",
+    "repository-url" : "https://github.com/fusesource/hawtjni",
+    "test-code-url" : "N/A",
+    "documentation-url" : "https://repo1.maven.org/maven2/org/fusesource/hawtjni/hawtjni-runtime/$version$/hawtjni-runtime-$version$-javadoc.jar",
+    "description" : "HawtJNI Runtime provides the annotations and helper API that projects using HawtJNI compile against. It includes the runtime classes used to describe JNI bindings and load bundled native libraries from Java.",
     "tested-versions" : [
       "1.11"
     ],

--- a/stats/org.fusesource.hawtjni/hawtjni-runtime/1.11/execution-metrics.json
+++ b/stats/org.fusesource.hawtjni/hawtjni-runtime/1.11/execution-metrics.json
@@ -1,0 +1,99 @@
+{
+  "fix_java_run_fail:2026-06-05": {
+    "timestamp": "2026-06-05T21:09:14.422402Z",
+    "library": "org.fusesource.hawtjni:hawtjni-runtime:1.11",
+    "starting_commit": "0ce85012240c4ee1e7d1edba0165eb5222e27361",
+    "ending_commit": "2a686f2037793efdf9b6218dd89bf0df22367030",
+    "previous_library": "org.fusesource.hawtjni:hawtjni-runtime:1.9",
+    "strategy_name": "java_run_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "1.11",
+      "dynamicAccess": {
+        "breakdown": {
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 228,
+          "missed": 1414,
+          "ratio": 0.138855,
+          "total": 1642
+        },
+        "line": {
+          "covered": 45,
+          "missed": 287,
+          "ratio": 0.135542,
+          "total": 332
+        },
+        "method": {
+          "covered": 14,
+          "missed": 44,
+          "ratio": 0.241379,
+          "total": 58
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "1.9",
+      "dynamicAccess": {
+        "breakdown": {
+          "resources": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 199,
+          "missed": 1408,
+          "ratio": 0.123833,
+          "total": 1607
+        },
+        "line": {
+          "covered": 43,
+          "missed": 284,
+          "ratio": 0.131498,
+          "total": 327
+        },
+        "method": {
+          "covered": 13,
+          "missed": 44,
+          "ratio": 0.22807,
+          "total": 57
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 21810,
+      "cached_input_tokens_used": 104448,
+      "output_tokens_used": 1236,
+      "iterations": 1,
+      "cost_usd": 0.0893,
+      "tested_library_loc": 45,
+      "metadata_entries": 1,
+      "previous_library_metadata_entries": 0,
+      "code_coverage_percent": 13.55,
+      "previous_library_coverage_percent": 13.15
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/src/test/java/org_fusesource_hawtjni/hawtjni_runtime/LibraryTest.java",
+      "metadata_file": "metadata/org.fusesource.hawtjni/hawtjni-runtime/1.11/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/org.fusesource.hawtjni/hawtjni-runtime/1.11/stats.json
+++ b/stats/org.fusesource.hawtjni/hawtjni-runtime/1.11/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "1.11",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "resources" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 1,
+          "totalCalls" : 1
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 1,
+      "totalCalls" : 1
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 228,
+        "missed" : 1414,
+        "ratio" : 0.138855,
+        "total" : 1642
+      },
+      "line" : {
+        "covered" : 45,
+        "missed" : 287,
+        "ratio" : 0.135542,
+        "total" : 332
+      },
+      "method" : {
+        "covered" : 14,
+        "missed" : 44,
+        "ratio" : 0.241379,
+        "total" : 58
+      }
+    }
+  } ]
+}

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/.gitignore
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/build.gradle
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "org.fusesource.hawtjni:hawtjni-runtime:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/gradle.properties
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = org.fusesource.hawtjni:hawtjni-runtime:1.11
+library.version = 1.11
+metadata.dir = org.fusesource.hawtjni/hawtjni-runtime/1.11/

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/settings.gradle
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'org.fusesource.hawtjni.hawtjni-runtime_tests'

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/src/test/java/org_fusesource_hawtjni/hawtjni_runtime/LibraryTest.java
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/src/test/java/org_fusesource_hawtjni/hawtjni_runtime/LibraryTest.java
@@ -1,0 +1,53 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_fusesource_hawtjni.hawtjni_runtime;
+
+import org.fusesource.hawtjni.runtime.Library;
+import org.junit.jupiter.api.Test;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class LibraryTest {
+    private static final String MISSING_LIBRARY_NAME = "hawtjni_runtime_coverage_probe_missing";
+
+    @Test
+    public void loadQueriesClasspathNativeLibraryResourcesWhenSystemLoadFails() {
+        RecordingClassLoader classLoader = new RecordingClassLoader();
+        Library library = new Library(MISSING_LIBRARY_NAME, null, classLoader);
+
+        UnsatisfiedLinkError error = assertThrows(UnsatisfiedLinkError.class, library::load);
+
+        assertThat(error).hasMessageContaining("Could not load library");
+        assertThat(classLoader.resourceNames()).containsExactly(
+                library.getPlatformSpecifcResourcePath(),
+                library.getOperatingSystemSpecifcResourcePath(),
+                library.getResorucePath());
+    }
+
+    private static final class RecordingClassLoader extends ClassLoader {
+        private final List<String> resourceNames = new ArrayList<>();
+
+        private RecordingClassLoader() {
+            super(null);
+        }
+
+        @Override
+        public URL getResource(String name) {
+            resourceNames.add(name);
+            return null;
+        }
+
+        private List<String> resourceNames() {
+            return resourceNames;
+        }
+    }
+}

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/src/test/java/org_fusesource_hawtjni/hawtjni_runtime/LibraryTest.java
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/src/test/java/org_fusesource_hawtjni/hawtjni_runtime/LibraryTest.java
@@ -28,6 +28,7 @@ public class LibraryTest {
 
         assertThat(error).hasMessageContaining("Could not load library");
         assertThat(classLoader.resourceNames()).containsExactly(
+                library.getArchSpecifcResourcePath(),
                 library.getPlatformSpecifcResourcePath(),
                 library.getOperatingSystemSpecifcResourcePath(),
                 library.getResorucePath());

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/user-code-filter.json
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/user-code-filter.json
@@ -1,0 +1,10 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "org.fusesource.hawtjni.runtime.**"
+    }
+  ]
+}

--- a/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/user-code-filter.json
+++ b/tests/src/org.fusesource.hawtjni/hawtjni-runtime/1.11/user-code-filter.json
@@ -5,6 +5,9 @@
     },
     {
       "includeClasses" : "org.fusesource.hawtjni.runtime.**"
+    },
+    {
+      "includeClasses" : "org_fusesource_hawtjni.hawtjni_runtime.**"
     }
   ]
 }

